src/app/core/migration/migration.service.ts

```diff
@@ -135,7 +135,7 @@ export class MigrationService {
   private _addProjectIdToEntity(
     pids: string[],
     entityProjectStates: { [key: string]: EntityState<any> },
-    additionalChanges: {} = {}
+    additionalChanges: Record<string, unknown> = {}
```

Comment:
Oh that's cool! I didn't even know about this one!

Reply:
haha, me neither! It was proposed as a solution in the resulting warnings description... and so I learned about it and used it ^^